subcmd/get.go

```diff
@@ -583,6 +583,9 @@ func (*getCmd) updateReposVersion(lockJSON *lockjson.LockJSON, reposPath pathuti
 		}
 		// Add repos to 'repos'
 		lockJSON.Repos = append(lockJSON.Repos, *repos)
+		sort.SliceStable(lockJSON.Repos, func(i, j int) bool {
+			return strings.ToLower(lockJSON.Repos[i].Path.FullPath()) < strings.ToLower(lockJSON.Repos[j].Path.FullPath())
```

Comment:
Why using `strings.ToLower()` ?
I think it is unnecessary because a URL is case-sensitive.

Reply:
I did it because the (hypothetical) entries

```
github.com/Andreas/somerepo
github.com/anton/someotherrepo
github.com/Berta/something
github.com/boris/anything
github.com/Clarissa/one
github.com/cora/two
```

would otherwise be sorted as

```
github.com/Andreas/somerepo
github.com/Berta/something
github.com/Clarissa/one
github.com/anton/someotherrepo
github.com/boris/anything
github.com/cora/two
```

which doesn't make much sense in my opinion.

However that URLs are normally case sensitive, but github repos are actually case insensitive is actually [a problem on its own](#277).

Reply:
I'm concerned that, for example, if two or more URLs can be differentiated only in case-insensitive, it always doesn't make a same result.

ex)
https://github.com/user/repo
https://github.com/user/Repo

It's rare case though, but it can occur in your code and it should not occur.
Yes, I suppose you think it's intuitive for users.
But I think URLs should not be distinguished by case-insensitive.

Reply:
> However that URLs are normally case sensitive, but github repos are actually case insensitive is actually [a problem on its own](#277).

Yes, true.
But volt supports (I want to support in the future) repositories not only GitHub.
It's better to be pessimistic for the future.

Reply:
> I'm concerned that, for example, if two or more URLs can be differentiated only in case-insensitive, it always doesn't make a same result.

Maybe I am missing something, but I used `sort.SliceStable` to have a predictable order of equal elements.

Reply:
`repos` entries in lock.json must be unique. But currently it just has a bug #288 ... :smirk:
If all elements are unique, it doesn't matter if sort is stable or not.

I'm creating the fix in #288.
So it's fixed soon later.